app/src/main/java/org/piwigo/helper/AccountHelper.java

```diff
@@ -30,14 +32,20 @@
 import org.piwigo.io.model.LoginResponse;
 import org.piwigo.ui.model.User;
 
-import java.util.Arrays;
+import java.util.ArrayList;
+import java.util.Collections;
 import java.util.List;
+import java.util.Observable;
 
 import javax.inject.Inject;
 import javax.inject.Singleton;
 
+/* TODO: cleanup this class. It has strange methcods... also the Userlist could be improved
+* Observer notifications to be documented */
+/* TODO: accounthelper should recognize if an account was deleted in the android system settings
+    and notify the observers accordingly */
 @Singleton
-public class AccountHelper {
+public class AccountHelper extends Observable {
```

Comment:
I plan to remove this class as it's a bit hacky.. See WIP #26

Reply:
That's fine for me, but don't forget to keep an eye on the social aspect collaborative development. - It is not very motivating for some willing to contribute if you destroy his work in such a direct way...